CHANGELOG.rst

```diff
@@ -6,6 +6,13 @@ Changelog
 Releases
 --------
 
+v2.2.2 (2020-08-20)
+~~~~~~~~~~~~~~~~~~~
+
+* Fix `determine_best_price` returning a spot price that would be below the current spot price in some conditions.
+* Dropped support for Python 3.5.
```

Comment:
@ksindi If you think this should be a major release, then I'd also be okay with releasing this as `v3.0.0`. I was debating whether this would be necessary as there are not many functional differences.